odoo/tools/config.py

```diff
@@ -124,6 +124,9 @@ def __init__(self, fname=None):
         group.add_option("--upgrade-path", dest="upgrade_path",
                          help="specify an additional upgrade path.",
                          action="callback", callback=self._check_upgrade_path, nargs=1, type="string")
+        group.add_option("--upgrade-scripts", dest="upgrade_scripts", my_default="",
```

Comment:
I find the name `--upgrade-scripts` confusing.
Other suggestions:
 - init-scripts
 - pre-update-scripts
 - setup-scripts

Reply:
I went with `pre-update-scripts`